serverless.yml

```diff
@@ -4,7 +4,7 @@ provider:
   name: aws
   runtime: nodejs6.10
   stage: v1
-  region: us-west-2
+  region: ap-southeast-2
```

Comment:
Shall we use Serverless Variables here ? https://github.com/serverless/serverless/pull/1834